### PR TITLE
feat(huobi) - pair create time

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/huobi.ts [hunk 1]
```diff
@@ -1699,6 +1699,13 @@ export default class huobi extends Exchange {
             // 7 Settlement Completed
             // 8 Delivered
             // 9 Suspending of Trade
+            let created = undefined;
+            let createdDate = this.safeString (market, 'create_date'); // i.e 20230101
+            if (createdDate !== undefined) {
+                const createdArray = this.stringToCharsArray (createdDate);
+                createdDate = createdArray[0] + createdArray[1] + createdArray[2] + createdArray[3] + '-' + createdArray[4] + createdArray[5] + '-' + createdArray[6] + createdArray[7] + ' 00:00:00';
+                created = this.parse8601 (createdDate);
```
Comment (carlosmiei) [inline ts/src/huobi.ts:1707]: same, not consistent

Comment (ttodua) [reply ts/src/huobi.ts:1707]: was this an issue? maybe you didn't mean here, because `parse8601` returns number, not string

Comment (carlosmiei) [reply ts/src/huobi.ts:1707]: sorry, yes didn't mean that, but second question do we really need this custom stringToCharsArray function? We have for instance `slice`

Comment (ttodua) [reply ts/src/huobi.ts:1707]: but `slice` with empty separator works only in JS and not transpilable, so in other langs we needed these base functions. if you can think of any other approach, lmk

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**